pkg/pdc/client.go

```diff
@@ -50,13 +50,18 @@ type Config struct {
 	// Used for local development.
 	// DevNetwork is the network that the agent will connect to.
 	DevNetwork string
+
+	// Used for local development.
+	// DevHost is the host that the agent will connect to.
+	DevHost string
 }
 
 func (cfg *Config) RegisterFlags(fs *flag.FlagSet) {
 	var deprecated string
 	fs.StringVar(&cfg.Token, "token", "", "The token to use to authenticate with Grafana Cloud. It must have the pdc-signing:write scope")
 	fs.StringVar(&cfg.HostedGrafanaID, "gcloud-hosted-grafana-id", "", "The ID of the Hosted Grafana instance to connect to")
 	fs.StringVar(&cfg.DevNetwork, "dev-network", "", "[DEVELOPMENT ONLY] the network the agent will connect to")
+	fs.StringVar(&cfg.DevHost, "dev-network", "localhost", "[DEVELOPMENT ONLY] the host the agent will connect to")
```

Comment:
`dev-network` is duplicated, did you mean `dev-host`?

```suggestion
	fs.StringVar(&cfg.DevHost, "dev-host", "localhost", "[DEVELOPMENT ONLY] the host the agent will connect to")
```

Reply:
heh yes definitely!